tests/utils/autoTester.js

```diff
@@ -12,8 +12,9 @@ TestCase = function (name, path, func) {
 }
 
 var currentTestCase = null;
+var usePrimaryCamera = false;
 
-module.exports.setupTest = function () {
+module.exports.setupTest = function (primaryCamera) {
```

Comment:
Why would you use the primary camera?  I had two problems with that.  The first was that the menu is on-screen, and may have been moved around.  The second was that I had someone testing with a monitor that had a different resolution (not HD), and the images looked different.

Reply:
Yes, I was aware of the resolution issues, so it wouldn’t always work with the image comparison, but it was needed to test 1. overlays and 2. the deferred rendering framebuffer debugging.